examples/01_prepare_data/mind_utils.ipynb

```diff
@@ -0,0 +1,528 @@
+{
```

Comment:
<p>Minor detail, would you mind changing the private function <code>_download_and_extract_globe</code>  to public <code>download_and_extract_globe</code>.  In other notebooks we don't import private functions</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/microsoft/recommenders/pull/1247/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='2'/>